app/controllers/events_controller.rb

```diff
@@ -4,22 +4,6 @@ class EventsController < InheritedResources::Base
   skip_load_and_authorize_resource only: [:index, :show]
```

Comment:
This commit and the previous one seem to belong together (Moving files and code). In such a case it makes sense to have the changes in one commit.
Please squash this commit into the previous one.